addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -0,0 +1,147 @@
+from odoo import api, fields, models, _
+from ..misc import date_diff, validate_dates
+
+
+class StockPickingExtraFields(models.Model):
+    """
+        Model to hold all extra fields of stock.picking object
+    """
+
+    _name = "udes_picking_extras"
+    _description = "UDES Picking Extra Fields"
+
+    picking_id = fields.Many2one("stock.picking", required=True, index=True, copy=False)
```

Comment:
Can we add `ondelete="cascade"`?

At present I think if the picking was deleted the extra fields record would stay in the database, and the extra fields record is not much use if not attached to a picking. :)

Reply:
done